webui/react/src/components/TaskConnectModal.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import CodeSample from 'hew/CodeSample';
+
+
+import { Label } from 'hew/Typography';
+import { Modal } from 'hew/Modal';
+
+
+export interface Props {
+    title?: string;
+    fields?: TaskConnectField[];
```

Comment:
qq: do these props need to be optional?

Reply:
fields should be required but title is set to default value if not provided. i removed optional type for fields